sdk/documentintelligence/tests.native.yml

```diff
@@ -12,5 +13,7 @@ extends:
     CloudConfig:
       Public:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+        ServiceConnection: azure-sdk-tests
+        SubscriptionConfigurationFilePath: eng/common/TestResources/sub-config/AzurePublicMsft.json
```

Comment:
Change this to:
          SubscriptionConfigurationFilePaths: 
            - eng/common/TestResources/sub-config/AzurePublicMsft.json

Reply:
updated guidance ^^